SharpBucket/V2/EndPoints/DiffParameters.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SharpBucket.V2.EndPoints
+{
+    /// <summary>
+    /// Parameters for RepositoryResource.GetDiff().
+    /// </summary>
+    public class DiffParameters: ParametersBase
+    {
+        /// <summary>
+        /// Number of context lines for each difference; default is 3.
+        /// </summary>
+        public byte Context { get; set; } = 3;
```

Comment:
In practical terms, no one is going to want more than 255 lines of context. But if you think a longer type is better, it should still be an unsigned type, since negative values don't make any sense.

Reply:
ok keep it as a byte then.